.travis.yml

```diff
@@ -1,18 +1,25 @@
 sudo: false
 language: d
-d:
-  - dmd-nightly
-  - dmd-beta
-  - dmd
-  - ldc-beta
-  - ldc
 
 os:
   - linux
-  - osx
 
 env:
-  - BUILD=
-  - BUILD=dub
+    matrix:
+        - BUILD=
+        - BUILD=dub
+    global:
+        - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
```

Comment:
Use the matrix `include` feature if you want to run this on a single build. Please don't kill all the other compiler tests.

Reply:
Yeah, this is not needed at all, all you need to do for packaging can be done in the stage that runs only on tags.